leakpro/attacks/mia_attacks/abstract_mia.py

```diff
@@ -256,3 +261,117 @@ def run_attack(self:Self) -> Union[AttackResult, List[AttackResult]]:
 
         """
         pass
+
+    def _memorization(self:Self) -> None:
```

Comment:
I am wondering if we should have this functionality in abstractMIA or if it makes more sense to have it in utils. Any opinion?

Reply:
Basically, we could create a .py file called memorization

Reply:
Done! The memorization process have been move to a file called "boosting.py". Here we can define more boosting techniques later, or have it open for others to do.